cmd/bridge/main.go

```diff
@@ -592,6 +592,16 @@ func main() {
 		},
 	}
 
+	go func() {
+		// Listen on port 8080 to be redirected to the console
+		redirectServer := http.NewServeMux()
+		redirectServer.HandleFunc("/", func(res http.ResponseWriter, req *http.Request) {
+			log.Infof("Redirecting from %s to %s", req.Host, listenURL.String())
+			http.Redirect(res, req, listenURL.String()+req.URL.Path, http.StatusMovedPermanently)
+		})
+		http.ListenAndServe(":8080", redirectServer)
```

Comment:
We should add a flag (and YAML config option) for the port and then only listen if the flag is set.

Reply:
We need to add error handling

Reply:
> We should add a flag (and YAML config option) for the port and then only listen if the flag is set.

That I kinda wanted to avoid with handling the redirect on the operator side.

Reply:
It shouldn't be much code and is a private API, so no openshift/api PR needed.